app/views/shared/_comment.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="comment">
+    <p><%= comment.author.name %>:</p>
+    <p><%= comment.text %></p>
+</div>
```

Comment:
- [ ] As the previous reviewer suggested that you don't have views files for **comments**. You should have a comments folder in your views files **(./app/views/comments/new.html.erb)** :blush:

Reply:
@HAMMAS-SALEEM @HAMMAS-SALEEM @codecaiine  in the shared file, I have comment html file, also in the previous code I had that file, the project is working 100% can you please point your finger to case where the project is not working, because in the requirements they only asked for the project to work they did not ask for specific file called new.html.erb when I already have file which was there and was for the same purpose

Reply:
Hey @shakerAbuDrais I hope you are doing well. Actually, he asked for a dedicated new.html.erb file Now you will ask how so when he said views/comments/**new.html.erb** So if you see in the comments folder there is a new.html.erb file :blush:

Reply:
@HAMMAS-SALEEM thank you, i fixed all now thank you again